.github/workflows/release.yaml

```diff
@@ -0,0 +1,39 @@
+# `name` value will appear "as is" in the badge.
+# See https://docs.github.com/en/actions/configuring-and-managing-workflows/configuring-a-workflow#adding-a-workflow-status-badge-to-your-repository
+# yamllint --format github .github/workflows/release.yaml
+---
+name: "release"
+
+on:
+  push:
+    tags: 'v[0-9]+.[0-9]+.[0-9]+**'  # Ex. v0.2.0 v0.2.1-rc2
```

Comment:
Shall you ass `^`?

Reply:
yeah, let's ass that!

Reply:
e64a8239a0ad03ef59e84328a29800347bde15e5